kits/hackathon-project-mentor/README.md

```
@@ -0,0 +1,99 @@
+
+## Tech Stack
+
+- Lamatic
+- Groq (LLM Provider)
+- Llama 3.3 70B (or compatible LLM)
+
+---
+
+## License
+
+This project is contributed as an AgentKit Template.
```

markdownlint-cli2 (0.23.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)